ga4gh/datamodel/__init__.py

```diff
@@ -187,3 +218,21 @@ def _scanDataFiles(self, dataDir, patterns):
             atexit.register(_cleanupHtslibsMess, indexDir)
         if numDataFiles == 0:
             raise exceptions.EmptyDirException(dataDir, patterns)
+
+    def getFileHandle(self, dataFile):
+        if dataFile in self.memo_table:
```

Comment:
If you're testing the cache itself, some low value should suffice, like 3.  But this value should not be the default for production.  Change `serverconfig` to set the various values.